code/datums/gamemodes/crash.dm

```diff
@@ -178,11 +178,12 @@
 	planet_nuked = CRASH_NUKE_INPROGRESS
 	INVOKE_ASYNC(src, .proc/play_cinematic, z_level)
 
-/datum/game_mode/infestation/crash/proc/on_nuke_started(obj/machinery/nuclearbomb/nuke)
+/datum/game_mode/infestation/crash/proc/on_nuke_started(datum/source, obj/machinery/nuclearbomb/nuke)
```

Comment:
is this only called thorugh signals?

Reply:
yes only via `RegisterSignal(SSdcs, COMSIG_GLOB_NUKE_START, .proc/on_nuke_started)`